<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-MB86S1A" failures="1" tests="1" name="Test(failed)" time="24.199" errors="0" timestamp="2023-04-23T15:40:31 IST">
  <testcase classname="TestCases.verifylogintestcase" name="login" time="0.642">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Please check your username . If you still can&amp;apos;t log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can&amp;apos;t log in, contact your Salesforce administrator.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Please check your username . If you still can't log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]
at Resources.commonutilities.handleAssertion(commonutilities.java:31)
at TestCases.verifylogintestcase.login(verifylogintestcase.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Test(failed) -->
